layers/tls.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2012 Google, Inc. All rights reserved.
+// Copyright 2009-2011 Andreas Krennmair. All rights reserved.
+// TLS decoder is a contribution from Jose Selvi
```

Comment:
Rather than these 3 lines, just add yourself as an author in the toplevel AUTHORS file, and say

// Copyright 2018 The GoPacket Authors.  All rights reserved.